bsp4j/src/ch/epfl/scala/bsp4j/BuildServerCapabilities.java

```diff
@@ -18,6 +18,8 @@ public class BuildServerCapabilities {
 
   private Boolean inverseSourcesProvider;
 
+  private Boolean wrappedSourcesProvider;
```

Comment:
This value is uninitialized, like all the others. Better use `boolean` for such simple on/off settings.